itou/templates/apply/includes/state_transition_siae.html

```diff
@@ -165,16 +165,16 @@ <h3 class="font-weight-normal text-muted">Détails du contrat de travail</h3>
             <hr>
 
             <h3 class="font-weight-normal text-muted">Rétractation</h3>
-            <p>
-                Vous pouvez vous rétracter jusqu'à {{ cancellation_days }} jours après le début du contrat, soit jusqu'au {{ job_application.cancellation_delay_end|date:"d/m/Y" }} inclus.
-            </p>
+
             {% if job_application.to_siae.is_subject_to_eligibility_rules %}
                 <p>
                     Si vous annulez cette embauche, vous ne pourrez pas prétendre à l'aide au poste pour les jours éventuellement travaillés.
                 </p>
                 <p>
                     {{ job_application.job_seeker.get_full_name|title }} restera éligible à l'IAE et pourra de nouveau vous envoyer une candidature dans le futur.
                 </p>
+                <p>
+                </p>Si une fiche salarié a été saisie pour cette embauche, elle sera annulée.</p>
```

Comment:
```suggestion
                <p>Si une fiche salarié a été saisie pour cette embauche, elle sera annulée.</p>
```

Enlève tes moufles ;) Faut faire gaffe quand même car une mauvaise hiérarchie de balises HTML peut casser une mise en page.